lib/main.coffee

```diff
@@ -32,3 +44,7 @@ module.exports =
 
     newColumns = undefined if newColumns.length is 0
     atom.config.set('wrap-guide.columns', newColumns)
+
+  toggle: ->
+    currentState = atom.config.get('wrap-guide.enabled')
```

Comment:
Fair point. I'm usually always a fierce supporter of consistency. I suggest we get @benogle and @lee-dohm to weigh in on this as well (and anyone else with an opinion).

/cc @atom/feedback